imap_processing/tests/spice/test_spin.py

```diff
@@ -97,6 +97,15 @@ def test_interpolate_spin_data(query_met_times, expected, fake_spin_data):
         )
 
 
+def test_get_spin_number(fake_spin_data):
```

Comment:
Parameterize over a few values so you can test a few different cases? (Same with repoint test possibly)
```suggestion
@pytest.mark.parameterize("met_time,spin_number", [(0, 0), (14.9, 0), (15.1, 1), (32, 2)]
def test_get_spin_number(fake_spin_data, met_time, spin_number):
```